cloud/amazon/ecs_service_facts.py

```diff
@@ -168,19 +170,19 @@ def list_services(self, cluster):
         if cluster and cluster is not None:
             fn_args['cluster'] = cluster
         response = self.ecs.list_services(**fn_args)
-        relevant_response = dict(services = response['serviceArns'])
-        return relevant_response
+        self.module.exit_json(services=response['serviceArns'])
 
     def describe_services(self, cluster, services):
         fn_args = dict()
         if cluster and cluster is not None:
             fn_args['cluster'] = cluster
         fn_args['services']=services.split(",")
         response = self.ecs.describe_services(**fn_args)
-        relevant_response = dict(services = map(self.extract_service_from, response['services']))
+        services = map(self.extract_service_from, response['services'])
+        services_not_running = None
         if 'failures' in response and len(response['failures'])>0:
-            relevant_response['services_not_running'] = response['failures']
-        return relevant_response
+            services_not_running = response['failures']
+        self.module.exit_json(services=services, services_not_running=services_not_running)
```

Comment:
Thanks for the info. My original thought was to leave the original output structure of the module intact (i.e. `services`, `services_not_running`). I am already proposing a breaking change, but this would be more major. Let me see if I can  create an example of the current state compared to the returning the boto3 response.